packages/gestalt/src/Masonry.js

```diff
@@ -29,7 +28,15 @@ type Layout =
   | LegacyMasonryLayout
   | LegacyUniformRowLayout;
 
-type Props<T> = {|
+export type Layer = {
```

Comment:
Could we use exact flow types? `{|` https://flow.org/en/docs/types/objects/#toc-exact-object-types

Reply:
done!